### ISSUE TITLE
Automated Requests via Web

### ISSUE THREAD

**Issue by KiTTYsh** [skaianet-radio/skaianet-web#2]
Automated requests should be able to be made via the website.

Requests should be limited so that people can not flood or spam the request queue.

**Comment by KiTTYsh** [comment]
The main interface and backend has been completed, but still needs spam protection and a few fixes before it can be made public.

**Comment by ganc** [comment]
Ability to mod the pending request list

**Comment by KiTTYsh** [comment]
Yeah, that would probably be good as well, which reminds me that we also need an administrative panel (created at issue #6).

Thus far, to be completed, the request system needs:
- Flood and spam protection
- Request list moderation
- Minor fixes and QA testing
